### DISCUSSION TITLE
Файлы игры

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2_classic#79] [General]
> Поделись файлом gothic.dat - буду смотреть, откуда тогда взялось 600 опыта в моих декомпилированных скриптах.

вот тут по ссылке в папке G2_or_de
https://yadi.sk/d/2nBogZV8FLYx4Q

**Comment by dosinabox** [comment]
Есть ли разница между Миненталем оригинала и аддона? За исключением табличек, которые вставляются в мир скриптом.

**Reply by Kor-Angar** [reply]
Мне кажется есть. Пытаюсь найти в чем, а найти не могу. Посмотри на зелье силы которое вставлено в пещере со склетом мракориса. Мне казалось что оно был но в другом месте.

**Comment by dosinabox** [comment]
Вспомнился алтарь Белиара в аддоне. А значит мир оригинала всё-таки нужно добавлять в мод.

**Comment by Kor-Angar** [comment]
> Вспомнился алтарь Белиара в аддоне. А значит мир оригинала всё-таки нужно добавлять в мод.

а файлы мира по размеру не совпадают ?

**Comment by dosinabox** [comment]
> а файлы мира по размеру не совпадают ?

В аддоне больше на примерно 200 кб.

**Comment by Kor-Angar** [comment]
Надо тестить. Проверить в первую очень перманентные растения (их в классике много) , потом эликсиры, потому шмотки. 17:48, 12 июля 2020 г., dosinabox <notifications@github.com>:

а файлы мира по размеру не совпадают ?

В аддоне больше на примерно 200 кб.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Отправлено из мобильного приложения Яндекс.Почты

**Comment by Kor-Angar** [comment]
У меня кстати игра вылетала один раз в Минентале, там где эликсир силы лежит17:48, 12 июля 2020 г., dosinabox <notifications@github.com>:

а файлы мира по размеру не совпадают ?

В аддоне больше на примерно 200 кб.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Отправлено из мобильного приложения Яндекс.Почты

**Comment by dosinabox** [comment]
> Надо тестить. Проверить в первую очень перманентные растения (их в классике много) , потом эликсиры, потому шмотки. 

Это неважно, потому что я в любом случае буду включать в мод мир оригинала.

**Reply by Kor-Angar** [reply]
Это было бы не важно если бы скрипты были оригинальные немецкие ;-)

**Comment by dosinabox** [comment]
А у тебя есть GOTHIC.DAT от английской 1.30?

**Reply by Kor-Angar** [reply]
Наверное ты про 1,31 Английская и русская 1,31
Немецкая 1,30.Поищу.

**Comment by Kor-Angar** [comment]
Немецкая 1,29
[exe_Gothic2_1_29_de.zip](https://github.com/dosinabox/g2_classic/files/6024387/exe_Gothic2_1_29_de.zip)
[Gothic2_1_29_de.zip](https://github.com/dosinabox/g2_classic/files/6024390/Gothic2_1_29_de.zip)

Немецкая 1,30
[exe_Gothic2_1_30_de.zip](https://github.com/dosinabox/g2_classic/files/6024389/exe_Gothic2_1_30_de.zip)
[Gothic2_1_30_de.zip](https://github.com/dosinabox/g2_classic/files/6024403/Gothic2_1_30_de.zip)

**Comment by dosinabox** [comment]
Английская 1.31 сойдет, если есть.

**Comment by Kor-Angar** [comment]
Пока ищу22:25, 22 февраля 2021 г., dosinabox <notifications@github.com>:
Английская 1.31 сойдет, если есть.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Отправлено из мобильного приложения Яндекс.Почты

**Comment by Kor-Angar** [comment]
![image](https://user-images.githubusercontent.com/49514764/108766392-2a328500-7566-11eb-8068-dbf3d03d9485.png)
а как такую подсветку сделать в Notepad++ ?:-)

**Reply by Kor-Angar** [reply]
> https://worldofplayers.ru/threads/41093/

**Comment by Kor-Angar** [comment]
не могу найти 1,31 вроде тут должен был лежать http://www.bendlins.de/nico/

но в веб архиве записей нет.

в целом 1,31 патч отличается от 1,30 только языком.

**Comment by dosinabox** [comment]
А в настоящей классике с дисков Акеллы переводили credits.bik и credits2.bik?

**Reply by dosinabox** [reply]
И какая там картинка при запуске? Приходится самодельные рисовать.

![image](https://user-images.githubusercontent.com/39790055/161448855-29cea90a-c7b9-4270-8257-3bb354be10ba.png)
![image](https://user-images.githubusercontent.com/39790055/161448860-f231f848-273e-4d9a-b610-d97753f3be61.png)

**Reply by Kor-Angar** [reply]
> А в настоящей классике с дисков Акеллы переводили credits.bik и credits2.bik?

нет. самая первая ссылка в топики там видео с дисков акеллы ;-)

**Reply by Kor-Angar** [reply]
> И какая там картинка при запуске? Приходится самодельные рисовать.

а она же непожатая должна лежать в папке DESKTOP ?

(https://github.com/dosinabox/g2_classic/files/8405643/NOMIP.zip)

**Reply by Kor-Angar** [reply]
или  это 
MENU_GOTHIC для Г2 и MENU_MASKE для г1 ?

**Reply by dosinabox** [reply]
SPLASH.BMP из самого Gothic2.exe. Появляется до запуска игры.

**Reply by Kor-Angar** [reply]
Gothic2.exe. с диска Акеллы
[Gothic2.zip](https://github.com/dosinabox/g2_classic/files/8405772/Gothic2.zip)

**Comment by Kor-Angar** [comment]
![image](https://user-images.githubusercontent.com/49514764/161451147-e83dc5cf-cf43-4f0f-b076-ea2bd5425f32.png)
![image](https://user-images.githubusercontent.com/49514764/161451152-995647c2-6166-4b9f-8849-4fe3b9ffa7cd.png)

**Comment by dosinabox** [comment]
А скинь gothic2.exe версии 1.3, пожалуйста.

**Reply by Kor-Angar** [reply]
если ты про немецкий экзешник (у Акеллы английский) то он в посте от [on 22 Feb 2021)

**Reply by dosinabox** [reply]
Это патчи. Мне нужен сам файл из игры.

**Reply by Kor-Angar** [reply]
этот? Gothic2.exe. с диска Акеллы
[Gothic2.zip](https://github.com/dosinabox/g2_classic/files/8405772/Gothic2.zip)

**Reply by dosinabox** [reply]
Этот, но у меня не получилось разобрать его на ресурсы. Он у тебя запускается? Какая картинка показывается до запуска игры?

**Reply by Kor-Angar** [reply]
я думаю тебе нужен кркянутый эзешник акеллы :-) я сейчас поищу

**Reply by Kor-Angar** [reply]
попробуй вот этот NoCD Gothic2
[Gothic2.zip](https://github.com/dosinabox/g2_classic/files/8413009/Gothic2.zip)

**Reply by dosinabox** [reply]
Спасибо)
![image](https://user-images.githubusercontent.com/39790055/161633608-e890848b-895b-4699-a332-1ba92053c890.png)

**Reply by Kor-Angar** [reply]
так значит это английский экзешник, а не крякнутый акеллы? :-)

**Reply by dosinabox** [reply]
Не знаю, меня только картинка интересует. Может, Акелла её не меняла. Может, крякнутый экзешник только английской версии. Можешь запустить некрякнутую Акеллу и проверить?

**Reply by Kor-Angar** [reply]
там же защита какая то стоит,  и игра у меня не запустится без системпака.

**Comment by Kor-Angar** [comment]
у меня еще такой файл в текстурах моей интернешенел сборки есть
[MENU_G2_ru.zip](https://github.com/dosinabox/g2_classic/files/8413213/MENU_G2_ru.zip)

**Reply by Kor-Angar** [reply]
это логотип в меню
![image](https://user-images.githubusercontent.com/49514764/161707796-d541a3f4-abc6-48e8-9dea-737b72ad2b2c.png)

**Comment by Kor-Angar** [comment]
сейчас попробую тут что нибудь найти https://disk.yandex.ru/d/wDo2xU3PK2JUnA?w=1

**Reply by Kor-Angar** [reply]
там нету

**Comment by Kor-Angar** [comment]
1.30 DE
[MAINDIR.zip](https://github.com/dosinabox/g2_classic/files/9280482/MAINDIR.zip)

**Comment by dosinabox** [comment]
Фигня какая-то. Я зачем-то добавляю в контент мода файл g2_classic_credits2.bik, идентичный существующему в стим-версии credits2.bik. В дисковой версии Г2НВ от Акеллы как-то иначе?

credits.bik - краткие титры (огонь в классике, залы Яркендара в аддоне)
credits2.bik - полные титры, одинаковые в классике и аддоне

**Reply by Kor-Angar** [reply]
> В дисковой версии Г2НВ от Акеллы как-то иначе?

НВ или классик?

**Reply by Kor-Angar** [reply]
**Акелла готик 2 классик**
![image](https://user-images.githubusercontent.com/49514764/195994064-94bbe314-de77-430a-91d9-3e96b85755c9.png)

**Reply by Kor-Angar** [reply]
**Акелла Адон**
![image](https://user-images.githubusercontent.com/49514764/195994136-d74d2a55-f551-417f-bfe8-a7659fb071d6.png)

**Reply by Kor-Angar** [reply]
пиратка 
![image](https://user-images.githubusercontent.com/49514764/195995041-1aa50ed0-d4f1-4e3d-9ed3-c0ef712c3ef4.png)

**Reply by Kor-Angar** [reply]
> credits.bik - краткие титры (огонь в классике, залы Яркендара в аддоне)

у Акеллы на диски похоже нет версии титров для Аддона :-)  титры на английском из Готики 2 классик на английском.

**Reply by dosinabox** [reply]
лишний ролик удален из версии для мастерской

**Comment by Kor-Angar** [comment]
манулы
[Gothic II Manual HU.zip](https://github.com/dosinabox/g2_classic/files/9792429/Gothic.II.Manual.HU.zip)
[Gothic II Manual SK.zip](https://github.com/dosinabox/g2_classic/files/9792430/Gothic.II.Manual.SK.zip)
[Gothic II Manual CZ.zip](https://github.com/dosinabox/g2_classic/files/9792432/Gothic.II.Manual.CZ.zip)
[Gothic II Manual English.zip](https://github.com/dosinabox/g2_classic/files/9792433/Gothic.II.Manual.English.zip)

**Reply by dosinabox** [reply]
HU? Знаю только фанатский перевод классики. Есть и официальный?

**Reply by Kor-Angar** [reply]
не понял. ты про что? :-)

**Reply by Kor-Angar** [reply]
понял. это какой то язык.

**Reply by Kor-Angar** [reply]
турецкий?

**Reply by dosinabox** [reply]
hungarian

**Reply by Kor-Angar** [reply]
венгерский

**Reply by Kor-Angar** [reply]
скрины в меню на английском. но это не показатель

**Comment by Kor-Angar** [comment]
а у тебя исходники sfx.data где лежат? :-)

**Reply by dosinabox** [reply]
Файл берется из обновления:
https://github.com/dosinabox/g2nr_unofficial_update/tree/master/PrjSFX

**Comment by dosinabox** [comment]
У тебя есть образ дисков Г2 1.3 от Акеллы?

**Reply by Kor-Angar** [reply]
да. там версия 1.31.
1.30 - немецкая
1.31 - англ
1.31 - русская

**Reply by dosinabox** [reply]
Да, 1.31 это хорошо. Можешь загрузить куда-нибудь, пожалуйста?

**Reply by Kor-Angar** [reply]
конечно. но там стоит защита. ты что именно хочешь оттуда? я по кускам оттуда уже всё загружал :-)

**Comment by Kor-Angar** [comment]
Gothic 2 пиратка 1.28

- https://disk.yandex.ru/d/nmrbaYEWtMhXrQ
- https://disk.yandex.ru/d/Sz8iONZoPhENkw

**Comment by Kor-Angar** [comment]
картинки можно было бы добавить в более высоком разрешение. у тебя в каком то отдельном томе по моему были
![20230219221358_1](https://user-images.githubusercontent.com/49514764/219975615-dbb56eda-a694-494a-9bd4-1c3709f49bc5.jpg)
![20230219223824_1](https://user-images.githubusercontent.com/49514764/219975623-c0996616-6b4f-41a6-93f2-98c1166dfd52.jpg)

**Reply by Kor-Angar** [reply]
https://steamcommunity.com/sharedfiles/filedetails/?id=3117339817

**Reply by dosinabox** [reply]
А зачем требовать мод классики?

**Reply by dosinabox** [reply]
добавила эти текстуры в русский контент классики

**Comment by Kor-Angar** [comment]
собрал файлы в архив что бы можно было запустить игру на движке 1.30 и потестировать без установки игры

[G2_130.zip](https://github.com/dosinabox/g2_classic/files/11509853/G2_130.zip)
